src/features/lightspeed/inlineSuggestions.ts

```diff
@@ -841,5 +840,84 @@ export async function inlineSuggestionUserActionHandler(
     inlineSuggestionFeedbackPayload,
     lightSpeedManager.orgTelemetryOptOut
   );
+  resetSuggestionData();
+}
+
+function inlineSuggestionPending(): boolean {
+  if (vscode.window.activeTextEditor?.document.languageId !== "ansible") {
+    return false;
+  }
+  const editor = vscode.window.activeTextEditor;
+  if (!editor) {
+    return false;
+  }
+  if (!inlineSuggestionData["suggestionId"]) {
+    return false;
+  }
+  return true;
+}
+
+function resetSuggestionData(): void {
   inlineSuggestionData = {};
+  insertTexts = [];
+}
+
+export async function rejectPendingSuggestion() {
+  const lightSpeedSettings =
+    lightSpeedManager.settingsManager.settings.lightSpeedService;
+  if (
+    suggestionDisplayed.get() &&
+    lightSpeedSettings.enabled &&
```

Comment:
This can be moved to a get function to check if lightspeed inline suggestion is enabled along with top level setting.

Reply:
Thank you for your comment.  I have simplified those checks keeping the logic identical in the latest commit.